src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.ts

```diff
@@ -150,9 +152,11 @@ export class InstructorAuditLogsPageComponent implements OnInit {
                 })),
             mergeAll(),
             finalize(() => this.isLoading = false))
-        .subscribe(((student: Students) =>
+        .subscribe(((student: Students) => {
+          const studentList: Student[] = [...student.students].sort(this.sortStudentList(SortBy.GIVER_NAME));
                 // Student with no name is selectable to search for all students since the field is optional
```

Comment:
Small issue, but perhaps a better name for function `sortStudentList`?  `sortStudentList` sounds like it will return a sorted list, and then it's sorted again which is a little weird.

Reply:
Yes, I see the redundancy now. Possible change to `compareStudentsLexicallyBy`  . It would read better and explain the intended ordering.